resources/views/media/manager.blade.php

```diff
@@ -293,7 +293,8 @@
 
                 <div class="modal-body">
                     <h4>{{ __('voyager::media.destination_folder') }}</h4>
-                    <select class="form-control">
+                    <select class="form-control" v-model="modals.move_files.destination">
+                        <option value="">Select Destination</option>
```

Comment:
Any specific reason for this option?
What happens if I use it?

Reply:
Mainly for placeholder text. When using v-model on the select input we have a default value, currently set to `''` and if this option is not there the select input appears empty, this prevents that and gives the user instruction.

Currently, if used it tries to move the file or folder to the same location it already exists which returns an error "File already exists at path." If you're okay with keeping this as a "placeholder" option, I'd like to update this PR by adding a check the value before submitting the post request to move the file.